ebpfcore/usersim/EbpfCore_Usersim.vcxproj

```diff
@@ -132,10 +132,6 @@
     <DebuggerFlavor>DbgengKernelDebugger</DebuggerFlavor>
   </PropertyGroup>
   <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Release|x64'">
-    <ResourceCompile>
-      <AdditionalIncludeDirectories>%(AdditionalIncludeDirectories);$(DDK_INC_PATH);$(SolutionDir)include</AdditionalIncludeDirectories>
-      <PreprocessorDefinitions>%(PreprocessorDefinitions);BINARY_COMPATIBLE=0;NT;UNICODE;_UNICODE;NDIS60;POOL_NX_OPTIN_AUTO</PreprocessorDefinitions>
```

Comment:
This one used to pass some different preprocessor definitions and the DDK include path - is it ok for it to inherit the common stuff and not get those?

Reply:
I guess a few of them passed these in - but it doesn't seem like they were used, so if you agree that they weren't used then it's great that they standardize on the same set of defines for resources for the whole repo.

Reply:
Yes, they appear to be unused.